daemon/ct.go

```diff
@@ -103,3 +103,26 @@ func (d *Daemon) EnableConntrackGC() {
 		}
 	}()
 }
+
+// CleanConntrack cleans the connection tracking table.
+func (d *Daemon) CleanConntrack() {
+	d.endpointsMU.RLock()
+
+	for k := range d.endpoints {
+		e := d.endpoints[k]
+		e.Mutex.RLock()
+		if e.Consumable == nil {
+			e.Mutex.RUnlock()
+			continue
+		}
+		e.Mutex.RUnlock()
+		// We can unlock the endpoint mutex sense
+		// in runGC it will be locked as needed.
+		runGC(e, true, 0xFFFF) // 0xFFFF is the max interval possible.
```

Comment:
Do I understand this correctly? You are expiring *all* entries of the conntrack table whenever policy is deleted. That is definitely not what we want. We want to expire CT entries which are related to policy rules which have been deleted.